code/common/pcapdecoder.q

```diff
@@ -0,0 +1,87 @@
+\d .pcap
```

Comment:
this file is in `code/common/` which means it will be loaded into (almost) every TorQ process - is this what is desired? It seems like it may only be required in the filealerter process, in which case it could be moved to `code/filealerter/` & make sure filealerter is configured to load process code (there's a config variable, I think it's `.proc.loadprocesscode:1b` or something similar)